### DISCUSSION TITLE
feat(slemicro): Add support for the SUSE Linux Enterprise Micro family

### DISCUSSION THREAD

**Discussion by msmeissn** [aquasecurity/trivy#7217] [Ideas]
### Description

SUSE is having more and more success with its SUSE Linux Enterprise Micro family and various rebuilds.

The data is exported already via the same CVRF as the SUSE Linux Enterprise Server one, so not much work would be needed.

I can help with implementation.

### Target

Container Image

### Scanner

Vulnerability

**Comment by knqyf263** [comment]
It would be appreciated if you work on it. Created https://github.com/aquasecurity/trivy/issues/7221 for tracking.
